src/sinks/elasticsearch/config.rs

```diff
@@ -123,6 +127,7 @@ pub struct ElasticsearchConfig {
         default
     )]
     #[configurable(derived)]
+    #[configurable(metadata(docs::advanced))]
     pub encoding: Transformer,
```

Comment:
I'm curious why the encoding (transformer) is advanced here, but nowhere else?

Reply:
That's a good question, and I'll double-check with Ari, but my _assumption_ is because the codec is actually fixed, so all that shows up under "encoding" are the transformer settings i.e. `only_fields`, `except_fields`, etc.

Those are, in my opinion as well, not very common.. and so marking them as advanced, by marking this whole field as advanced, seems reasonable.